.github/workflows/e2e-tools.yml

```diff
@@ -45,7 +45,7 @@ jobs:
 
     - name: Build tools
       run: |
-        make build-tools binary-nrocli
+        make build-tools binary-nrovalidate
```

Comment:
it is a tool which supposes to validate, so maybe ```nrovalidator``` is more accurate here?

Reply:
I don't have strong feelings. We're not following too closely the unix conventions anyway, so if you want it renamed I can do it.